build.gradle

```diff
@@ -178,10 +178,10 @@ List<PublishedProject> publishedProjects = [
 
 import org.jetbrains.dokka.gradle.DokkaTask
 tasks.withType(DokkaTask).configureEach {
-    moduleName.set("Conclave Core")
+    moduleName.set("Conclave")
     dokkaSourceSets {
         register("conclave") {
-            displayName.set("Conclave Core")
+            displayName.set("Conclave")
```

Comment:
Can you try reverting this back to Conclave Core. I want the title in the API docs to say that.

Reply:
Reverted.
Will raise another PR to fix the broken links.

Reply:
No, just revert the display name. The other `moduleName` _probably_ refers to the link so change that to Conclave.

Reply:
Doesn't work. The moduleName controls both the title and the links!